AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -652,6 +654,24 @@
 "settings.security.title" = "Security";
 "settings.security.file.protection" = "File Protection";
 "settings.security.file.protection.description" = "Encrypts files stored on this device when it is locked.";
+"settings.security.passcode.lock" = "Passcode Lock";
+"settings.security.passcode.lock.description" = "Your Alfresco account details, synced files and local files will be wiped after 10 failed passcode attempts.";
```

Comment:
This string should be built at runtime using the `kRemainingAttemptsMaxValue` constant to avoid having to update the strings if we decide to change the number of attempts.